Orange/evaluation/scoring.py

```diff
@@ -287,46 +286,35 @@ def compute_score(self, results, eps=1e-15, normalize=True,
             dtype=np.float64, count=len(results.probabilities))
 
 
-## Regression scores
+# Regression scores
 
-class MSE(Score):
+class MSE(RegressionScore):
     __wraps__ = skl_metrics.mean_squared_error
-    class_types = (ContinuousVariable, )
     long_name = "Mean square error"
 
-    def compute_score(self, results):
-        return self.from_predicted(results, skl_metrics.mean_squared_error)
 
-
-class RMSE(Score):
-    class_types = (ContinuousVariable, )
+class RMSE(RegressionScore):
     long_name = "Root mean square error"
 
     def compute_score(self, results):
         return np.sqrt(MSE(results))
 
 
-class MAE(Score):
+class MAE(RegressionScore):
     __wraps__ = skl_metrics.mean_absolute_error
-    class_types = (ContinuousVariable, )
     long_name = "Mean absolute error"
 
 
-    def compute_score(self, results):
-        return self.from_predicted(results, skl_metrics.mean_absolute_error)
-
-
 # pylint: disable=invalid-name
-class R2(Score):
+class R2(RegressionScore):
     __wraps__ = skl_metrics.r2_score
-    class_types = (ContinuousVariable, )
     long_name = "Coefficient of determination"
 
     def compute_score(self, results):
```

Comment:
This method is removed in the next comment. (Adding by chunks didn't work properly. :))